GameData/ROEngines/PartConfigs/Agena_SSTU.cfg

```diff
@@ -275,5 +277,239 @@ PART
+	
+	tags = agena, model8096, agena-2000, 2000
+	
+	engineType = Agena
+	stagingIcon = LIQUID_ENGINE
+	stageOffset = 1
+    childStageOffset = 1
+	
+	TechRequired = orbitalRocketry1986
+	cost = 200
+	entryCost = 7000
+	
+	MODULE
+	{
+		name = ModuleEnginesRF
+		thrustVectorTransformName = thrustTransform
+	}
+	
+	MODULE
+	{
+		name = ModuleGimbal
+		gimbalResponseSpeed = 15
+		useGimbalResponseSpeed = true
+		gimbalTransformName = yawGimbal
+		gimbalRange = 3
+
+		gimbalRangeXN = 0
+		gimbalRangeXP = 0
+		gimbalRangeYN = 3
+		gimbalRangeYP = 3
+	}
+		MODULE
+	{
+		name = ModuleGimbal
+		gimbalResponseSpeed = 15
+		useGimbalResponseSpeed = true
+		gimbalTransformName = pitchGimbal
+		gimbalRangeXN = 3
+		gimbalRangeXP = 3
+		gimbalRangeYN = 0
+		gimbalRangeYP = 0
+	}
+	MODULE
+	{
+		name = FXModuleLookAtConstraint
+
+		CONSTRAINLOOKFX
+		{
+			targetName = gimbal1_Upper
+			rotatorsName = gimbal1_Lower
+		}
+		CONSTRAINLOOKFX
+		{
+			targetName = gimbal1_Lower
+			rotatorsName = gimbal1_Upper
+		}
+		CONSTRAINLOOKFX
+		{
+			targetName = gimbal2_Upper
+			rotatorsName = gimbal2_Lower
+		}
+		CONSTRAINLOOKFX
+		{
+			targetName = gimbal2_Lower
+			rotatorsName = gimbal2_Upper
+		}
+	}
+	MODULE
+	{
+		name = FXModuleThrottleEffects
+		fxModuleNames = throttleColor
+		responseSpeed = 1.0
+		dependOnEngineState = True
+		dependOnThrottle = True
+	}
+	MODULE
+	{
+		name = ModuleColorChanger
+		moduleID = heatColor
+		animRate = 0.025
+		shaderProperty = _EmissiveColor
+		excludedRenderer = innerGlow
+		toggleInEditor = false
+		toggleInFlight = false
+		redCurve
+		{
+			key = 1 1 1 1
+			key = 1 1 1 1
+		}
+		greenCurve
+		{
+			key = 0 0 1 1
+			key = 1 1 1 1
+		}
+		blueCurve
+		{
+			key = 0 0 1 1
+			key = 1 1 1 1
+		}
+		alphaCurve
+		{
+			key = 0 0 1 1
+			key = 1 1 1 1
+		}
+	}
+	MODULE
+	{
+		name = ModuleColorChanger
+		moduleID = throttleColor
+		animRate = 1
+		shaderProperty = _TintColor
+		includedRenderer = innerGlow
+		toggleInEditor = false
+		toggleInFlight = false
+		redCurve
+		{
+			key = 0 0
+			key = 1 1
+		}
+		greenCurve
+		{
+			key = 0 0
+			key = 1 1
+		}
+		blueCurve
+		{
+			key = 0 0
+			key = 1 1
+		}
+		alphaCurve
+		{
+			key = 0 0
+			key = 1 1
+		}
+	}
+}
+
+@PART[ROE-Agena8096C]:AFTER[RealismOverhaulEngines]
+{
+	@title = XLR81 Agena Model 8096C Vacuum Engine
+	@MODULE[ModuleEngineConfigs] 
+	{
+		@configuration = Model8096L
+		!CONFIG[XLR81-BA-5] {}
+		!CONFIG[XLR81-BA-7] {}
+		!CONFIG[XLR81-BA-11] {}
+		!CONFIG[XLR81-BA-13] {}
+		!CONFIG[Model8096-39] {}
+		!CONFIG[Model8096A] {}
+		!CONFIG[XLR81-LF2-SPS] {}
+	}
+}
+//TU support for Agena 8096C
```

Comment:
Please move this with the rest of the TU Support so they are all in one place.

Reply:
Fixed